tasks/client_keys.yml

```diff
@@ -67,3 +67,23 @@
     dest: "{{ openvpn_fetch_config_dir }}/{{item}}/{{inventory_hostname}}{{config_fetch_suffix | default('')}}.ovpn"
     flat: yes
   with_items: "{{clients}}"
+  
+- name: remove client keys and csr
+  file:
+    path: "{{ openvpn_key_dir }}/{{ item }}.key"
+    state: absent
```

Comment:
If we delete the files, they'll get recreated on every role run because the files aren't present (`command` looks at the `creates` file to decide whether or not to run).
Potential solution: Delete and recreate the files (`touch`)?